services/cd-service/pkg/repository/repository.go

```diff
@@ -3223,6 +3223,33 @@ func (s *State) GetApplicationTeamOwner(ctx context.Context, transaction *sql.Tx
 	}
 }
 
+func (s *State) GetAllApplicationsTeamOwner(ctx context.Context, transaction *sql.Tx) (map[string]string, error) {
+	result := make(map[string]string)
+	if s.DBHandler.ShouldUseOtherTables() {
+		apps, err := s.DBHandler.DBSelectAllAppsMetadata(ctx, transaction)
+		if err != nil {
+			return result, fmt.Errorf("could not get team of all apps: %w", err)
+		}
+		for _, app := range apps {
+			result[app.App] = app.Metadata.Team
+		}
+		return result, nil
+	} else {
+		apps, err := s.GetApplications(ctx, transaction)
+		if err != nil {
+			return result, err
+		}
+		for _, app := range apps {
+			teamOwner, err := s.GetApplicationTeamOwnerFromManifest(app)
+			if err != nil {
+				return result, err
+			}
+			result[app] = teamOwner
+		}
+		return result, nil
+	}
```

Comment:
this code will never be run in the cd-service, please remove it.

Reply:
We still have some tests that use this part of the code